pkg/controller/csi_handler_test.go

```diff
@@ -34,6 +34,7 @@ import (
 	"k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
 	core "k8s.io/client-go/testing"
+	csitranslator "k8s.io/csi-translation-lib"
```

Comment:
Can we add a test case for ReconcileVA using a volume type that does support migration?

Reply:
[These](https://github.com/kubernetes/kubernetes/blob/ded1f58779d0d2570735a6ebe7428d57a100ca97/staging/src/k8s.io/csi-translation-lib/translate.go#L29) are the ones that truly support in the actual translator. Do you suggest unit tests based on this? (or) we could setup a fake translator that migration support. If it's more work, can we do this separately?

Reply:
I think adding a unit test using one of the supported translated types would be good.